.agents/skills/repo-source-code-navigate/SKILLS.md

```
@@ -78,24 +78,30 @@ is no test directory.
 pnpm install                    # Install dependencies
 pnpm test                       # Run Jest tests (all packages)
-pnpm lint                       # ESLint + tsc + deno check (all packages)
-pnpm format                     # Format code with Prettier (all packages)
+pnpm check                      # Biome check (all packages)
+pnpm check:fix                  # Biome check and fix fixable issues (all packages)
 pnpm build                      # Build for publishing (all packages)
 ```
 
+They work and do the same thing from the root of the repository for all
+workspaces or from a workspace directory.
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)